docs/development.rst

```diff
@@ -73,7 +97,7 @@ contrib module, or notebooks, and so instead to test the core codebase a develop
 
 .. code-block:: console
 
-    pytest --ignore tests/benchmarks/ --ignore tests/contrib --ignore tests/test_notebooks.py
+    nox --session tests --python 3.10
```

Comment:
In full transparency, if you run this at the moment you will get a failure on `test_public_api_repr.py` but you get that if you run the tests without `nox` locally as well. It doesn't show up in CI though, so I'm not sure what's up with that. :?